### PR TITLE
Merge master into preview

### PR DESCRIPTION
**Pull request by timtay-microsoft**

### CODE REVIEW

File: vsts/gatedBuild.ps1 [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	{
+		Write-Host "Will skip Iot Hub tests"
+		$runTestCmd += " -skipIoTHubTests"
+	}
+	else 
+	{
+		Write-Host "Will run Iot Hub tests"
+	}	
+	
+	#if (ShouldSkipPnPTests)
```
Comment (abhipsaMisra) [inline vsts/gatedBuild.ps1]: uncomment this

Comment (timtay-microsoft) [reply vsts/gatedBuild.ps1]: This is deliberately commented out. We want to skip all PnP tests for now, regardless of what changes are made

Comment (abhipsaMisra) [reply vsts/gatedBuild.ps1]: Are you sure? I remember updating the pipeline manually a couple of days pack to pick up PnP E2E tests
@prmathur-microsoft  to confirm.

Comment (abhipsaMisra) [reply vsts/gatedBuild.ps1]: Did we mean that we should disable the tests that are failing/ all PnP E2E tests?

**Pull request merged**